net-mail/vpopmail/vpopmail-5.4.33-r5.ebuild

```diff
@@ -0,0 +1,253 @@
+		vchkpw.c vconvert.c vdelivermail.c \
+		vpopbull.c vpopmail.c vqmaillocal.c \
+		vuserinfo.c maildirquota.c || die
+
+	# remove vpopmail advertisement
+	sed -i -e '/printf.*vpopmail/s:vpopmail (:(:' \
+		vdelivermail.c vpopbull.c vqmaillocal.c || die
+
+	# automake/autoconf
+	mv -f "${S}"/configure.{in,ac} || die
+	sed -i -e 's,AM_CONFIG_HEADER,AC_CONFIG_HEADERS,g' \
+		configure.ac || die
+
+	# _FORTIFY_SOURCE
+	sed -i \
+		-e 's/\(snprintf(\s*\(LI->[a-zA-Z_]\+\),\s*\)[a-zA-Z_]\+,/\1 sizeof(\2),/' \
+		vlistlib.c || die
+
+	eautoreconf
+	ht_fix_file cdb/Makefile
+}
+
+src_configure() {
+	local authopts
+	if use mysql; then
+		incdir=$(mysql_config --variable=pkgincludedir || die)
+		libdir=$(mysql_config --variable=pkglibdir || die)
+		authopts+=" --enable-auth-module=mysql"
+		authopts+=" --enable-incdir=${incdir}"
+		authopts+=" --enable-libdir=${libdir}"
+		authopts+=" --enable-sql-logging"
+		authopts+=" --enable-valias"
+		authopts+=" --disable-mysql-replication"
+		authopts+=" --enable-mysql-limits"
+	elif use postgres; then
+		libdir=$(pg_config --libdir || die)
+		incdir=$(pg_config --pkgincludedir || die)
+		authopts+=" --enable-auth-module=pgsql"
+		authopts+=" --enable-incdir=${incdir}"
+		authopts+=" --enable-libdir=${libdir}"
+		authopts+=" --enable-sql-logging"
+		authopts+=" --enable-valias"
+	else
+		authopts+=" --enable-auth-module=cdb"
+	fi
+
+	econf ${authopts} \
+		--sysconfdir=${VPOP_HOME}/etc \
+		--enable-non-root-build \
+		--enable-qmaildir=${QMAIL_HOME} \
+		--enable-qmail-newu=${QMAIL_HOME}/bin/qmail-newu \
+		--enable-qmail-inject=${QMAIL_HOME}/bin/qmail-inject \
+		--enable-qmail-newmrh=${QMAIL_HOME}/bin/qmail-newmrh \
+		--enable-vpopuser=vpopmail \
+		--enable-vpopgroup=vpopmail \
+		--enable-many-domains \
+		--enable-file-locking \
+		--enable-file-sync \
+		--enable-md5-passwords \
+		--enable-logging \
+		--enable-auth-logging \
+		--enable-log-name=vpopmail \
+		--enable-qmail-ext \
+		--disable-tcpserver-file \
+		--disable-roaming-users \
+		$(use_enable ipalias ip-alias-domains) \
+		$(use_enable clearpasswd clear-passwd) \
+		$(use_enable maildrop) \
+		$(use_enable maildrop maildrop-prog /usr/bin/maildrop) \
+		$(use_enable spamassassin)
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
+	keepdir "${VPOP_HOME}"/domains
+
+	# install helper script for maildir conversion
+	into "${VPOP_HOME}"
+	dobin "${FILESDIR}"/vpopmail-Maildir-dotmaildir-fix.sh
+	into /usr
+
+	mv doc/doc_html/ doc/man_html/ . || die
+	einstalldocs
+	rm -r "${D}/${VPOP_HOME}"/doc || die
+	dosym \
+		$(realpath --relative-to "${D}/${VPOP_HOME}"/ "${D}"/usr/share/doc/${PF}/ || die) \
+		"${VPOP_HOME}"/doc
+
+	# create /etc/vpopmail.conf
+	if use mysql; then
+		insinto /etc
+		newins "${D}${VPOP_HOME}"/etc/vpopmail.mysql vpopmail.conf
+		dosym \
+			$(realpath --relative-to "${D}/${VPOP_HOME}"/etc/ "${D}"/etc/vpopmail.conf || die) \
+			"${VPOP_HOME}"/etc/vpopmail.mysql
+
+		sed -e '12d' -i "${D}"/etc/vpopmail.conf || die
```

Comment:
That is a horrible idea. It's going to silently start doing the wrong thing when the file changes for some reason.